app/views/coronavirus_pages/_sub_sections_summary_list.html.erb

```diff
@@ -13,9 +14,28 @@
      }
    }
  end %>
-<div class="covid-manage-page__accordion-summary-list">
-  <%= render "govuk_publishing_components/components/summary_list", {
-    title: @coronavirus_page.sections_title,
-    items: sub_sections
-  } %>
+
+<% if reorder  %>
+  <div class="covid-manage-page__accordion-summary-list">
+    <ul class="govuk-list step-by-step-reorder__list" id="js-reorder-group">
+      <% sub_sections.each_with_index do |accordion, index| %>
```

Comment:
`accordion` is a display choice and not a description of the object. I think it would be clearer if `sub_section` was used here. As that way there will be consistency in the way an instance of `SubSection` is labelled.

Also each `sub_section` now has a position, so we don't need to create an index to uniquely identify each subsection. So I think this line should become:
```irb
<% sub_sections.each do |sub_section| %>
```
and line 23 would become:
```irb
<div class="govuk-grid-row" id="step-<%= sub_section['position'] %>">
```

Reply:
I tried removing the index and using the sub_section position, but that didn't work. I think the problem is that a sub_section may not have a position initially.